### PR TITLE
[D1] Add python example

### PR DESCRIPTION
**Pull request by elithrar**
Adds a Python-based example for D1.

### CODE REVIEW

File: content/d1/examples/query-d1-from-python-workers.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+Support for Python in Cloudflare Workers is in beta. Review the [documentation on Python support](/workers/languages/python/) to understand how Python works within the Workers platform.
+
+{{</Aside>}}
+
+## Prerequisites
+
+Before getting started, you should:
+
+1. Review the [D1 tutorial](/d1/get-started/) for TypeScript and JavaScript to learn how to **create a D1 database and configure a Workers project**.
+2. Visit the [Python language guide](/workers/languages/python/) to understand how Python support works on the Workers platform.
```
Comment (Maddy-Cloudflare) [inline content/d1/examples/query-d1-from-python-workers.md]:
```suggestion
2. Refer to the [Python language guide](/workers/languages/python/) to understand how Python support works on the Workers platform.
```

File: content/d1/examples/query-d1-from-python-workers.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+# ┌───────────────────┬──────────────────────────────────────┐
+# │                   │ c89db32e-83f4-4e62-8cd7-7c8f97659029 │
+# ├───────────────────┼──────────────────────────────────────┤
+# │ name              │ db-enam                              │
+# ├───────────────────┼──────────────────────────────────────┤
+# │ created_at        │ 2023-06-12T16:52:03.071Z             │
+# └───────────────────┴──────────────────────────────────────┘
+```
+
+### Configure bindings
```
Comment (Maddy-Cloudflare) [inline content/d1/examples/query-d1-from-python-workers.md]:
```suggestion
### 1. Configure bindings
```

File: content/d1/examples/query-d1-from-python-workers.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+
+[[d1_databases]]
+binding = "DB" # This will be how you refer to your database in your Worker
+database_name = "YOUR_DATABASE_NAME"
+database_id = "YOUR_DATABASE_ID"
+```
+
+The value of `binding` is how you will refer to your database from within your Worker. If you change this, you must change this in your Worker script as well.
+
+### Create your Python Worker
```
Comment (Maddy-Cloudflare) [inline content/d1/examples/query-d1-from-python-workers.md]:
```suggestion
### 2. Create your Python Worker
```

File: content/d1/examples/query-d1-from-python-workers.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+    # Do anything else you'd like on request here!
+
+    # Query D1 - we'll list all tables in our database in this example
+    results = await env.DB.prepare("PRAGMA table_list").all()
+    # Return a JSON response
+    return Response.json(results)
+
+```
+
+Ensure that the value of `binding` in your `wrangler.toml` file exactly matches the name of the variable in your Python code. In this example, we refer to our database via a `DB` binding, and query it via `await env.DB.prepare(...)`.
```
Comment (Maddy-Cloudflare) [inline content/d1/examples/query-d1-from-python-workers.md]:
```suggestion
The value of `binding` in your `wrangler.toml` file exactly must match the name of the variable in your Python code. This example refers to the database via a `DB` binding, and query this binding via `await env.DB.prepare(...)`.
```

File: content/d1/examples/query-d1-from-python-workers.md [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+# Published python-and-d1 (0.51 sec)
+#   https://python-and-d1.YOUR_SUBDOMAIN.workers.dev
+# Current Deployment ID: 80b72e19-da82-4465-83a2-c12fb11ccc72
+```
+
+Your Worker will be available at `https://python-and-d1.YOUR_SUBDOMAIN.workers.dev`.
+
+If you receive an error deploying:
+
+* Make sure you have configured your `wrangler.toml` with the `database_id` and `database_name` of a valid D1 database
```
Comment (Maddy-Cloudflare) [inline content/d1/examples/query-d1-from-python-workers.md]:
```suggestion
* Make sure you have configured your `wrangler.toml` with the `database_id` and `database_name` of a valid D1 database.
```

File: content/d1/examples/query-d1-from-python-workers.md [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+* That you have set `compatibility_flags = ["python_workers"]` in your `wrangler.toml`, which is required for Python
```
Comment (Maddy-Cloudflare) [inline content/d1/examples/query-d1-from-python-workers.md]:
```suggestion
* Ensure `compatibility_flags = ["python_workers"]` is set in your `wrangler.toml`, which is required for Python.
```

File: content/d1/examples/query-d1-from-python-workers.md [hunk 7]
```diff
@@ -0,0 +1,127 @@
...
+* You have reviewed the [list of error codes](/workers/observability/errors/) and ensured your code does not throw an uncaught exception.
```
Comment (Maddy-Cloudflare) [inline content/d1/examples/query-d1-from-python-workers.md]:
```suggestion
* Review the [list of error codes](/workers/observability/errors/), and ensure your code does not throw an uncaught exception.
```

File: content/d1/examples/query-d1-from-python-workers.md [hunk 8]
```diff
@@ -0,0 +1,127 @@
...
+
+## Next steps
+
+* Visit the [documentation on Python Workers](/workers/languages/python/) to learn more about how to use Python in Workers.
```
Comment (Maddy-Cloudflare) [inline content/d1/examples/query-d1-from-python-workers.md]:
```suggestion
* Refer to [Workers Python documentation](/workers/languages/python/) to learn more about how to use Python in Workers.
```

**Review by Maddy-Cloudflare — APPROVED** [review]

**Comment by elithrar** [comment]
Thanks for the quick review @Maddy-Cloudflare!

**Pull request merged**